dowhy/causal_refuters/placebo_treatment_refuter.py

```diff
@@ -203,13 +216,6 @@ def refute_placebo_treatment(
         identified_estimand.instrumental_variables = [
             "placebo_" + s for s in identified_estimand.instrumental_variables
```

Comment:
why is this IV code removed? is it redundant?

Reply:
good catch, I removed it by mistake.